src/organisms/cards/RadioCard/index.js

```diff
@@ -123,6 +147,12 @@ RadioCard.propTypes = {
   image: PropTypes.string,
 
   /**
+   * optional icon displayed below description; `icon` overrides `image` prop;
+   * This is the icon name from the [Icon component](/#icon).
+   */
+  icon: PropTypes.string,
```

Comment:
we might also want to allow the user to pass in the height and width props for the icon component?